Yr/Yr.php

```diff
@@ -310,8 +335,10 @@ private static function downloadData($url, $path, $cacheLife)
             if (!empty($xml_content)) {
                 // Only cache if there is content from request
                 file_put_contents($path, $xml_content);
+                return $xml_content;
```

Comment:
This function should return a string. $xml_content is a resource (line 333). Just remove the line, the contents of the file will be returned from line 341 :)